Utest/test_geometry/testClassObstacleBoost.cpp

```diff
@@ -26,30 +26,18 @@
  *
  **/
 
-#define BOOST_TEST_MODULE ObstacleTest
-#define BOOST_TEST_DYN_LINK
 #include <boost/test/unit_test.hpp>
-#include "../../geometry/Obstacle.h"
-#include "../../geometry/Point.h"
-#include "../../geometry/Wall.h"
-#include "../../IO/OutputHandler.h"
+#include "geometry/Obstacle.h"
+#include "geometry/Point.h"
+#include "geometry/Wall.h"
```

Comment:
this change of path works because of the previous `target_link_libraries` in L701?

Reply:
Yes, cmake will add libraries and include paths for all targets that use a target if the respective properties are set. This makes it trivial to relocate libraries.